.vib/keycloak/cypress/cypress/integration/keycloak_spec.js

```diff
@@ -0,0 +1,87 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows login/logout', () => {
+  cy.login();
+  cy.get('#input-error').should('not.exist');
+  cy.get('.dropdown-toggle').click({ force: true });
+  cy.contains('Sign Out').click();
+  cy.contains('#kc-login', 'Sign In');
+});
+
+it('allows changing user data', () => {
+  cy.login();
+  cy.get('.dropdown-toggle').click({ force: true });
+  cy.contains('a', 'Manage account').click({ force: true });
+  cy.contains('p', 'Account console loading').should('not.exist'); //Wait until the loading is finished to continue with actions
```

Comment:
Do you really need the `'p'` part? I don't think there are any other elements in the UI with the same loading msg. Removing it (if possible) makes it less error-prone.